DroneInventorySystem/src/main/java/com/digitalojt/web/form/StockInfoForm.java

```diff
@@ -0,0 +1,26 @@
+package com.digitalojt.web.form;
+
+import com.digitalojt.web.validation.StockInfoFormValidator;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * 在庫一覧画面のフォームクラス
+ * 
+ * @author haruka matano
+ *
+ */
+@Data
+@StockInfoFormValidator
+/*追記箇所*/
+@Getter
+@Setter
+public class StockInfoForm 
+{
```

Comment:
```suggestion
@NotEmpty(message = "Category cannot be empty")
```
アノテーションを追加
メッセージは良い感じにリファクタリングしてください
他のフィールドにも追加